src/containers/MasterWindow.js

```diff
@@ -63,23 +63,28 @@ class MasterWindowContainer extends Component {
   async onWebsocketEvent(event) {
     const { includedTabsInfo, stale } = event;
 
+    // case: the whole document is staled
     if (stale) {
```

Comment:
One problem I see with that is the fact that with websocket events we get both `stale` and `includedTabsInfo` when just one row was edited. Not sure if that was the plan

![Screen Shot 2020-04-01 at 14 52 53](https://user-images.githubusercontent.com/513460/78139327-9d9c4700-7428-11ea-80e8-a0406845e3ce.png)

Reply:
I think I've solved it now, i.e. we are refreshing only the root document (header).